.github/workflows/front-tests.yml

```
@@ -38,10 +22,29 @@ jobs:
 
     - run: flutter --version
 
+    - name: Restore pub cache
+      uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)